CHANGES

```diff
@@ -1,4 +1,5 @@
 2.0~alpha
+* Changed license to LGPL 2.1 with linking exception, like OCaml
```

Comment:
I think it would make sense to mention the PR URL in this change entry, to give a justification for the change and provide traceability for the agreement process.